main/mainTemplate.json

```diff
@@ -8,7 +8,7 @@
         "artifactsBaseUrl": "/",
         "description": "The base URL for dependent assets"
       },
-      "defaultValue": "https://raw.githubusercontent.com/infobloxopen/infoblox-azure-templates/master/main/"
+      "defaultValue": "https://raw.githubusercontent.com/nsharda-infoblox/infoblox-azure-templates/863-template/main/"
```

Comment:
modify this to master

Reply:
Hi Kapil, yes, template testing is still in progress. Needed this to check the template. Will change back to master.

Will add label of Do No Review for now. I'll update PR when I am finished with testing.